editor/src/components/common/shared-strategies/convert-to-flex-strategy.ts

```diff
@@ -49,6 +56,12 @@ export function convertLayoutToFlexCommands(
       ]
     }
 
+    const childrenPaths = MetadataUtils.getChildrenPathsUnordered(metadata, path).flatMap((child) =>
+      isElementNonDOMElement(metadata, allElementProps, child)
+        ? replaceContentAffectingPathsWithTheirChildrenRecursive(metadata, allElementProps, [child])
```

Comment:
AFAIU we should only recursively replace non-dom elements here, and let the code convert sizeless divs

Reply:
added in https://github.com/concrete-utopia/utopia/pull/3557/commits/e596d1de34ca3052b97c0cdf10e834799b64850a